source/material_model/depth_dependent.cc

```diff
@@ -0,0 +1,444 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#include <aspect/material_model/depth_dependent.h>
+#include <deal.II/base/parameter_handler.h>
+#include <utility>
+#include <limits>
+using namespace dealii;
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+
+    namespace internal
+    {
+      class RadialViscosityLookup
+      {
+        public:
+          RadialViscosityLookup(const std::string &filename)
+          {
+            /* This method is used for the Table method of depth dependent viscosity */
+            std::string temp;
+            std::ifstream in(filename.c_str(), std::ios::in);
+            AssertThrow (in,
+                         ExcMessage (std::string("Couldn't open file <") + filename));
+
+            min_depth=DBL_MAX;
+            max_depth=DBL_MIN;
```

Comment:
The c++ way is to use `std::numeric_limits<double>::max()` and `-std::numeric_limits<double>::max()`.